src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/stars';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  font-style: normal;
+  color: $main-text-color;
```

Comment:
The variable '$main-text-color' is used here for the color property. Ensure that this variable is defined in the 'variables' file or imported properly to avoid compilation errors.